Library/ViewModels/PaymentMethodsViewModelTests.swift

```diff
@@ -174,6 +176,31 @@ internal final class PaymentMethodsViewModelTests: TestCase {
     }
   }
 
+  func testCancelLoadingState_Success() {
+    let response = UserEnvelope<GraphUser>(me: userTemplate)
+    let apiService = MockService(
+      fetchGraphUserResult: .success(response)
+    )
+
+    withEnvironment(apiService: apiService) {
+      self.cancelLoadingState.assertDidNotEmitValue()
+
+      self.vm.inputs
+        .shouldCancelPaymentSheetAppearance(state: false)
+
+      self.scheduler.advance()
+
+      self.cancelLoadingState.assertDidNotEmitValue()
+
+      self.vm.inputs
+        .shouldCancelPaymentSheetAppearance(state: true)
+
+      self.scheduler.advance()
+
+      self.cancelLoadingState.assertDidEmitValue()
+    }
+  }
```

Comment:
Would it be useful to add tests to ensure that the `addCardButton` is is/isn't hidden or interact-able at the right times? i.e. when`featureSettingsPaymentSheetEnabled()`

Reply:
The only time the Add New Card button isn't interactable is when the loading wheel is showing, but its' hard to get at the actual `UIButton` itself, as it is the `tableView'`'s `footerView` which is `private` to the view controller.